### DISCUSSION TITLE
"Normalizing" Credit Card Payments counting twice

### DISCUSSION THREAD

**Discussion by Fmstrat** [firefly-iii/firefly-iii#9359] [Firefly III]
Hi all, this is probably a basic organizational thing I'm missing, but here's my issue. Let's say I have these accounts and balances:

- Credit Card Asset Account: 0.00
- Checking Account: +1000.00

And these transactions:

- From Credit Card to a store (CC => no name): 100 = (CC @ -100.00)
- From Credit Card to another store (CC => no name): 100 = (CC now @ -200.00)
- From Checking to pay credit card (Checking => no name):  200 = (Checking now @ 800.00)
- To Credit Card (no name => CC): 200 = (CC @ 0.00)

Those last two are separate transactions, because they come from separate CSV imports. I don't mind this at all, but the problem is on the Dashboard my "In/Out" thinks that I'm out 400.00 instead of 200.00, because it's considering the two store transactions and the payment to the CC as "outs".

What is the proper way to handle this? Ideally I could convert those two transactions into a transfer from Checking to CC, but that doesn't seem to be an option, and some of the Issues in GitHub point to certain reasons why, but I'm not sure what to do otherwise.

Thanks!

**Comment by JC5** [comment]
The proper way would be to make sure that the money from the checking account to the credit card is a transfer.

If you delete transaction nr. 4 and convert transaction nr. 3 to a transfer, it should work as expected.
